go/client/cmd_simplefs.go

```diff
@@ -50,6 +50,11 @@ func NewCmdSimpleFS(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Comm
 func makeSimpleFSPath(g *libkb.GlobalContext, path string) keybase1.Path {
 	mountDir := "/keybase"
 
+	// remove trailing separator while preserving platform specificity
+	if strings.HasSuffix(path, "\\") || strings.HasSuffix(path, "/") {
```

Comment:
I'm not a big fan of peppering ("\" or "/") checks all throughout the code.  Can we, early on, convert all paths to Unix paths, and then only deal with one of these cases? (BTW, I thought we were already doing this?)